BlazarUI/app/scripts/models/BranchDefinition.js

```diff
@@ -0,0 +1,16 @@
+import Model from './Model';
+
+class BranchDefinition extends Model {
+
+  constructor(gitInfo) {
+    super()
+    this.gitInfo = gitInfo;
+  }
+
+  url() {
+    let gitInfo = this.gitInfo;
+    return `${config.apiRoot}/branch/lookup?host=${gitInfo.url}&organization=${gitInfo.org}&repository=${gitInfo.repo}&branch=${gitInfo.branch}`;
```

Comment:
need to keep these URLs consistent -- can you remove the trailing slash before branch?